java/server/src/org/openqa/grid/selenium/GridLauncherV3.java

```diff
@@ -230,7 +230,7 @@ private boolean parse(String[] args, Object options, CommonCliOptions common) {
     }
 
     configureLogging(common.getLog(), common.getDebug());
-    log.info(version());
+    log.finest(version());
```

Comment:
This change means that users can't easily see which version of the selenium server they're using. This is `info` level information.